### ISSUE TITLE
Translation service for documents

### ISSUE THREAD

**Issue by wfdd** [openpatata/openpatata#21]
Manually translating all documents into English and Turkish would be an impossible task, but translations would still be nice to have.  I suppose the only thing we can do is use a translation service like Google's or Bing's, but I don't think those two are free.

**Comment by akuckartz** [comment]
The European Commission does provide such services. As far as I remember they can be used for free (free as in beer).

**Comment by wfdd** [comment]
Thanks. It appears that you need to be a public servant to be allowed access (http://ec.europa.eu/dgs/translation/translationresources/machine_translation/index_en.htm).

**Comment by akuckartz** [comment]
You could explain the intentions of the project to MT@EC: This is about translating administration documents for the general public. Then they can decide to grant access or not. They can not grant permission if they are not asked (and if they deny they still might be able to provide helpful suggestions). You could also contact your local administration and ask for suggestions on how to proceeed.

**Comment by wfdd** [comment]
Yeah, I'll do that when the project continues. Thanks for your contributions.

**Comment by wfdd** [comment]
I've had a change of heart and sent them an email. For full disclosure:

> Dear MT@EC,
> 
> I'm a Cypriot citizen who has created openpatata (http://openpatata.eu/), a
> Cypriot-parliament tracker; that is, a website that attempts to bring the
> actions of the Cypriot House of Representatives to the attention of a wider
> audience.  As part of this effort, I've been aiming to translate several kinds
> of official documents, such as explanatory notes and committee reports, into
> English.  This would benefit English-speaking expats, as well as Turkish
> Cypriots and other non-Greek minorities.  I'm wondering whether I could be
> permitted to use your machine translation service for this purpose.  I should
> add that, for the reason that this is a project I have undertaken on my own,
> without any income, I cannot afford a paid translation service, like Google's or
> Bing's.
> 
> Regards
> 
> [redacted: my name]

And their response:

> Dear Sir, Madam,
> 
> Access to the European Commission Machine Translation Service is granted to Public Administrations in the EU Member States only via official domain email addresses, and not to personal email addresses such as hotmail, gmail, etc.
> If you work for an EU Public Administration please register in ECAS with your personal official domain email address and inform us by email, by including your full signature and clearly stating your functions and Public Administration you work for.
> 
> Thank you for your understanding,
> Best regards!
> The MT@EC Team

**Comment by akuckartz** [comment]
Thanks for trying this. So you need someone with a public mail address associated with a public administration. Someone working for a small municipality would be ok if I understand that correctly.

**Comment by wfdd** [comment]
I suppose it depends on if they have any other criteria for e-mail addresses; local governments aren't offered `gov.cy` addresses. It might also be that they only deal with national and federal governments.

**Comment by wfdd** [comment]
I brought this to the attention of a Cypriot MEP who's part of the petitions committee (should that make any difference).

> Dear Mr Hadjigeorgiou,
> 
> In anticipation of a negative response to my request to MT@EC (quoted below), I
> appeal to you, in your capacity as an MEP, to investigate the possibility of
> access to this tool being granted to members of the public—on a case-by-case
> basis, or otherwise.  I explain my predicament in my first e-mail to MT@EC, at
> the bottom.  I will be at your disposal should you find this to be a cause
> worthy of your time and require further clarification.
> 
> The MT@EC website is located at
> http://ec.europa.eu/dgs/translation/translationresources/machine_translation/index_en.htm.
> 
> Regards
> 
> [redacted: my name]
> 
> > Dear MT@EC,
> >
> > Yes, I do appreciate that those are the rules; no, I'm not a public servant.
> > I'm asking if an exception could be made, considering that it concerns the
> > translation of official government documents.  Alternatively, might there be a
> > similar service for public use?
> >
> > Regards
> >
> > [redacted: my name]
> >
> > > [...]
